.github/workflows/promote-config.yml

```diff
@@ -13,7 +13,10 @@ jobs:
     name: Open promotion pull request
     if: ${{ github.event.label.name == 'ok-to-promote' }}
     runs-on: ubuntu-latest
```

Comment:
/me likes Fedora 

would the benefits here just be minimizing risk?

Reply:
Ubuntu 24.04 has git v2.43.0, which is still behind Fedora 41. I actually didn't verify this would fix https://github.com/coreos/fedora-coreos-config/pull/3302#issuecomment-2545817324 but I don't want to keep debugging this before at least having it match the git version we're all using locally to test this.

v2.43.0 might turn out to be new enough, but I'm also not too interested in keeping track of Ubuntu repos to save a few seconds.

Reply:
Works as well 👍🏻